code/modules/spells/spell_types/pointed/duffelbag_curse.dm

```diff
@@ -24,8 +24,12 @@
 		return FALSE
 	if(!can_target(targets[1], user))
 		return FALSE
-
+	
 	var/mob/living/carbon/target = targets[1]
+	if(HAS_TRAIT(target, TRAIT_DUFFEL_CURSED))
+		to_chat(user, "<span class='warning'>That person is already cursed!</span>")
+		return FALSE
```

Comment:
why not just check to see if they have a cursed duffel bag on their person?

or just remove the code that tries to put the duffel bag in your hands if it can't fit onto your back

Reply:
Is that actually better or worse? Doesn't this basically accomplish the same thing?

And I'd rather it do something other than nothing if their back is full. This is quite edgecase for the most part, since I think only admemery or snails would result in becoming immune to duffelbags on your back at that point.